source/_integrations/edl21.markdown

```diff
@@ -29,10 +29,15 @@ To set it up, add the following information to your `configuration.yaml` file:
 ```yaml
 sensor:
   - platform: edl21
+    name: "Heat pump"
```

Comment:
In the first example we only want required config variables

Reply:
I removed it.